worlds/overcooked2/__init__.py

```diff
@@ -584,6 +584,7 @@ def kevin_level_to_keyholder_level_id(level_id: int) -> Optional[int]:
             "TwoStars": star_threshold_scale * 0.75,
             "OneStar": star_threshold_scale * 0.35,
         }
+        base_data["AlwaysServeOldestOrder"] = self.options.always_serve_oldest_order.result
```

Comment:
where did this suddenly come from?

Reply:
Was missing this option to send to the game